cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -101,5 +101,5 @@ def test_order_catalog_item(catalog_item, request):
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
         fail_func=requests.reload, num_sec=1400, delay=20)
-    assert row.last_message.text == 'Service Provisioned Successfully'
+    assert row.request_state.text == 'Finished'
```

Comment:
Are we going to assert somewhere else that the provision actually finished successfully? Finished != Success